webpack.config.babel.js

```diff
@@ -56,7 +65,7 @@ if (ENV === 'production') {
 
 const configDist = {
   context: `${__dirname}/src`,
-  entry: './index.js',
+  entry: ["babel-polyfill", './index.js'],
```

Comment:
I've used `babel-polyfill` because it allows me to use some functionalities that are  normally not available on IE11. It would fix errors like `'Symbol' is undefined` and I would also be able to use methods like `includes` for arrays etc. 26% size increase is a lot, maybe I can look for a different solution